sdk/samples/iot/paho_iot_hub_pnp_component_sample.c

```diff
@@ -57,11 +57,11 @@ static az_span pnp_device_components[] = { AZ_SPAN_LITERAL_FROM_STR("thermostat1
 static int32_t const pnp_components_length
     = sizeof(pnp_device_components) / sizeof(pnp_device_components[0]);
 
-// IoT Hub Device Twin Values
-static az_span const twin_reported_serial_number_property_name
+// IoT Hub Device Property Values
```

Comment:
Nit - i'm not sure this is really `IoT Hub Device property` since it's not the Hub twin anymore.  Could this be PnP property??